services/src/main/java/org/exoplatform/addons/gamification/rest/ChallengeRest.java

```diff
@@ -238,7 +245,11 @@ public Response getAllChallengesByUser(
     RuleFilter filter = new RuleFilter();
     filter.setTerm(term);
     filter.setUsername(currentUser);
-
+    if (StringUtils.isBlank(dateFilterType)) {
+      filter.setDateFilterType(DateFilterType.STARTED);
```

Comment:
Are you sure that default value should be `STARTED` instead of `ALL` ? IMO, it should be `ALL` for API and `STARTED` in UI only